web/extension/build/js/lv0.js

```diff
@@ -39,4 +39,4 @@
                 console.error("Critical error loading Ruffle into page")
             }
         }
-    }());
\ No newline at end of file
+    }());
```

Comment:
Please avoid unrelated changes in commits

Reply:
I think I edited lv0.js when debugging, and git added a new line to the end of the file(For some reason the javascript sources were missing newlines at the ends of the files).